server/services/educationAndWorkPlanService.test.ts

```diff
@@ -45,4 +47,36 @@ describe('educationAndWorkPlanService', () => {
       expect(actual).toEqual(Error('Service Unavailable'))
     })
   })
+
+  describe('getActionPlan', () => {
+    it('should get Action Plan', async () => {
+      // Given
+      const prisonNumber = 'A1234BC'
+      const userToken = 'a-user-token'
+      const actionPlan = aValidActionPlanDtoWithOneGoal()
+      educationAndWorkPlanClient.getActionPlan.mockImplementation(() => Promise.resolve(actionPlan))
+
+      // When
+      await educationAndWorkPlanService.getActionPlan(prisonNumber, userToken)
+
+      // Then
+      expect(educationAndWorkPlanClient.getActionPlan).toHaveBeenCalledWith(prisonNumber, userToken)
+    })
```

Comment:
Should we also be asserting the return value of the function?
IE. should line 60 be:
```
const actual = await educationAndWorkPlanService.getActionPlan(prisonNumber, userToken)
```
And then we make some assertions about `actual` ?

Reply:
Ok, I'll that in. I wasn't too bothered as it's just testing what's returned from the mock above, but I guess you could argue we need it for regression purposes 👍